langlib/lang.map/src/main/java/org/ballerinalang/langlib/map/RemoveAll.java

```diff
@@ -41,6 +42,7 @@
 
     public static void removeAll(Strand strand, MapValue<?, ?> m) {
         checkIsMapOnlyOperation(m.getType(), "removeAll()");
+        throwErrorOnRecord(m.getType(), "removeAll()");
```

Comment:
This is valid for some records. E.g record {| T...; |}, record {| T x?; |}

Reply:
Fixed